freedomofpress/securedrop-dev-docs
# Initial split of development documentation off end user docs

## Status

Ready for review

## Description of Changes

In scope of this PR:
- Removes non-development docs and assets
- Adds a light bit of structure to the development docs
- Updates links accordingly
- Simplifies versioning/build strategy

Not in scope:
- Major organizational changes or additions

## To test the modified Docker container

1. Run `docker build -f deploy/Dockerfile .` from the repository root.
2. If the image builds successfully, run `docker run -p 127.0.0.1:5080:5080 <image_id>` to spin up a container.
3. Visit the running container using `localhost:5080` in your browser.
4. Confirm that the documentation is built correctly, and that you can access the generated PDF file.
